examples/stream/stream.mjs

```diff
@@ -0,0 +1,103 @@
+import {
+  Writable,
+  Readable,
+  Duplex
+} from 'stream';
+
```

Comment:
All this files seems good utility that could live in a module.

Reply:
Yeah, I'll eventually make it a separate module. Want to harden it up a bit more and work on some benchmarking first.